### DISCUSSION TITLE
rabbitmq-server-3.11.2 Installation auf Windows 11 - Unrecognized option neighbours

### DISCUSSION THREAD

**Discussion by MHofmann-rossmanith** [rabbitmq/rabbitmq-server#6581] [Other]
Systeminformations:
- Windows 11 Pro Version 21H2 (Build No. 22000.1281)
- Erlang: otp_win64_25.1.2.exe
- RabbitMQ: rabbitmq-server-3.11.2.exe

Error:
When running the RabbitMQ Server installation with default settings, an error occurs while installing the RabbitMQ service.
![grafik](https://user-images.githubusercontent.com/56957131/205892571-914d863a-7333-4fa0-9cb5-69826613095f.png)

erl_crash.dump: [erl_crash.zip](https://github.com/rabbitmq/rabbitmq-server/files/10165499/erl_crash.zip)

**Comment by czthiele** [comment]
Hi @MHofmann-rossmanith,

I recently encountered the same error message (Unrecognized option neighbours), although not on Windows 11. With the help of this [comment](https://github.com/rabbitmq/rabbitmq-server/discussions/6104#discussioncomment-3953902) I could solve it. I dont kow if you are familiar with the erlang.cookie, but Erlang under Windows is respecting HOMEDRIVE/HOMEPATH to look for this file. Installation failed for me when my home drive was not accessible. I changed the HOMEDRIVE/HOMEPATH environment variable in the registry under _HKEY_CURRENT_USER\Volatile Environment_ to an accessible drive, and installation worked after that. I think this only needs to be done once when installing the service.
Maybe this can help in some way.

**Reply by keivankashani** [reply]
Thanks for your guidance, this solution does work

**Comment by lukebakken** [comment]
@MHofmann-rossmanith this is an odd error. Please open an administrative command prompt and run this command:

```
set > rabbitmq-server-6581.txt
```

Attach the generated file to your response. Then, can you please download this file:

https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.11.5/rabbitmq-server-windows-3.11.5.zip

...extract it, then, using an administrative command prompt, do the following:

```
cd path\to\rabbitmq_server-3.11.5
.\sbin\rabbitmq-server.bat
```

Please note if RabbitMQ starts correctly. If not, copy the output and attach it to your response. Then

```
.\sbin\rabbitmq-service.bat install
```

See if that command installs the service. Thanks
